src/main/resources/db/migration/V1.12__alter_employee_table.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE employees
+    ADD COLUMN email_address VARCHAR(100) NOT NULL,
+    ALTER COLUMN user_id DROP NOT NULL;
```

Comment:
why are you removing user_id from employees entity?

Reply:
I am not removing user_id,  it is the not null constraint i am dropping since at the point of creating employee, there is no user created

Reply:
You can create the user entity first and set the employee with that user entity in the same method